sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/ClientMetricsTest.java

```diff
@@ -84,6 +88,43 @@ public void afterTest() {
         this.meterRegistry = null;
     }
 
+    @Test(groups = { "unit" }, timeOut = TIMEOUT)
+    public void MaxValueExceedingDefinedLimitStillWorksWithoutException() throws Exception {
```

Comment:
nit -  name should start with lower case :)

Reply:
Fixed